src/components/marginals/Navbar/DesktopNav.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+
+import { NavSection, StyledHamburger, Logo } from './nav.styles';
+// eslint-disable-next-line import/no-useless-path-segments
+import { MenuContext, Container, Body } from '../../.';
+
+function DesktopNav() {
+  const menuContext = useContext(MenuContext);
+
+  const { toggleMenuOpen, menuOpen } = menuContext;
+
+  return (
+    <NavSection>
+      <Container>
+        <nav className='navWrapper'>
+          <div className='navLeft'>
+            <Logo src='brand.png' alt='Brand Logo' />
+          </div>
+
+          <div className='navRight'>
+            <StyledHamburger menuOpen={menuOpen} onClick={toggleMenuOpen} />
+
```

Comment:
You can create a single Array of objects containing the navbar details and then map them in the navlist.
2 benefits.

1. No repetitive code.
2. Navigation is going to be same for desktop and mobile components. So having a single source would be better so that we wont have to update at 2 places.

Reply:
Done